web/src/games/soupofletters/board.tsx

```diff
@@ -0,0 +1,228 @@
+import React from 'react';
+// game shared import 
+import { GameLayout } from 'gamesShared/components/fbg/GameLayout';
+import { IGameArgs } from 'gamesShared/definitions/game';
+import { isOnlineGame, isAIGame, isLocalGame } from 'gamesShared/helpers/gameMode';
+import { IPlayerInRoom } from 'gamesShared/definitions/player';
+import { PlayerBadges } from 'gamesShared/components/badges/PlayerBadges';
+import { Ctx } from 'boardgame.io';
+// node module import
```

Comment:
Remove comment

Reply:
well i though it woul be nice to comment what is coming from where